.github/workflows/deploy.yml

```diff
@@ -0,0 +1,30 @@
+name: deploy
+
+on:  
+  push:
+    tags:
+      - '*'
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/master'
```

Comment:
I'm presuming we'd only tag if we need to release - so in the off chance the tag was a release branch or something like that, may as well publish it.

Reply:
So remove this if check?

Reply:
I think so, especially if @ben-manes agrees :)

Reply:
Makes sense

Reply:
Done.